Telegram/SourceFiles/history/view/history_view_message.cpp

```diff
@@ -1858,8 +1860,9 @@ int Message::resizeContentGetHeight(int newWidth) {
 	accumulate_min(contentWidth, maxWidth());
 	_bubbleWidthLimit = std::max(st::msgMaxWidth, monospaceMaxWidth());
 	if (!AdaptiveBubbles()) {
-		accumulate_min(contentWidth, _bubbleWidthLimit);
-	}
+		if (MonospaceLargeBubbles()) {
+			accumulate_min(contentWidth, _bubbleWidthLimit);
+		}	}
```

Comment:
Please fix the indentation here:
```suggestion
		}
	}
```